build/README.md

```diff
@@ -140,27 +128,23 @@ which ever provider you choose.
     Kubernetes cluster containing the certificates and keys generated in the
     previous step.
 
-        ./apply-certs.sh <cluster_context>
+        ./apply-certs.sh <cluster_context> <namespace>
```

Comment:
If namespace is an optional argument to make-certs, it should be an optional argument here too (and both should use the same default value)

Reply:
Opting for making it required throughout.

Reply:
Looks good, but let's maintain stylistic consistency and put `<NAMESPACE>` in all caps.

Reply:
IMO capitalizing it doesn't mesh stylistically with `<cluster_context>`.

Reply:
I think the most important thing is to have the same thing represented the same way all the way through, so `<NAMESPACE>` or $NAMESPACE or `<namespace>` are all fine, but it should be just one of those.  I'm not particularly bothered by the capitalization of cluster_context not matching because it's a different kind of entity (folder name rather than arbitrary name), but I'd be happy capitalizing it to match.  Definitely `<cluster_context>` should match `<your_cluster_context>` above, but that's a pre-existing issue.